CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [v0.1.4] - 2023-04-TBD
+
+### Fixed
+
+- The F3 and F8 keyboard shortcuts (to Run and Run selected) are no longer working #230, #231
```

Comment:
Can we add links to the issue/pull request? Also, we may want to update the language to specify what was fixed, rather than what is wrong

Reply:
I noticed the numbers resolve as links when you view the CHANGELOG tab for the extension in VS Code and in the MarketPlace (https://marketplace.visualstudio.com/items/SAS.sas-lsp/changelog), so I think we are good there.

As for the language, yeah, probably better to take more positive voice -- I'll update. I was debating though, if at some point the CHANGELOG were to be data driven, it would probably come from the PR or issue title. I just copied the issue title.

Reply:
Yeah, I think you're right. If it comes from the PR, that _should_ be using the correct phrasing ("fix ...", "add ..."), but I've seen issues be less consistent with that